.github/workflows/ci.yml

```diff
@@ -75,11 +75,6 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-  tauri:
```

Comment:
Hmm, I think removing this from `ci.yml` entirely means these no longer run on PRs, right? I think that is a bit much, we want these to run on PRs, otherwise we don't know if they build correctly.

Reply:
Got a little trigger happy :-). I'll fix that.

Reply:
https://github.com/firezone/firezone/pull/7801